package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.1.3-fp-ver-2",
   "description": "Performance focused timeline for react",
   "private": false,
+  "types": "./types/index.d.ts",
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
-    "build_lib": "babel src --out-dir lib --ignore demo*,setupTests.js,*.test.js --copy-files --source-maps inline",
+    "build": "tsc && cp src/types.d.ts types/types.d.ts && webpack --config webpack.prod.js",
+    "build_lib": "babel src --out-dir lib -x '.js,.jsx,.mdx,.html,.tsx' --ignore **/demo.js,**/demo_index.js,**/demo.html,**/demo/,**/setupTests.js,**/*/*.test.js,**/stories/ --copy-files --source-maps inline --no-copy-ignored",
```

Comment:
cred ca lipseste inca ceva ca sa mearga complet build + deploy. Mai precis, 

* la `build_lib` vad ca nu e rulat tsc; ar trebui
* la `build`: cred ca (teoretic) nu trebuie tsc; iar acest script noi nu prea il folosim, nu? noi lucram in modul storybook, nu?
* cred ca trebuie facut un script "tsc" care doar apeleaza "tsc". Si as zice ca trebuie invocat de build, build_lib, storybook

Reply:
Da, build-ul nu il folosim si lucram in modul storybook.